Compilation Error with Rizin 0.7.3: rz_analysis_get_fcns Not Declared

Error message:
```
rz-retdec/src/rz-plugin/data.cpp: In member function ‘void retdec::rzplugin::RizinDatabase::fetchFunctionsAndGlobals(retdec::config::Config&) const’:
rz-retdec/src/rz-plugin/data.cpp:164:21: error: ‘rz_analysis_get_fcns’ was not declared in this scope; did you mean ‘rz_analysis_get_fcn_in’?
  164 |         auto list = rz_analysis_get_fcns(_rzcore.analysis);
      |                     ^~~~~~~~~~~~~~~~~~~~
      |                     rz_analysis_get_fcn_in

```
This is due to the function `rz_analysis_get_fcns` got removed in Rizin 0.7.3 (rizinorg/rizin#4353)

--- a/src/rz-plugin/data.cpp
+++ b/src/rz-plugin/data.cpp
@@ -161,7 +161,7 @@ Function RizinDatabase::fetchSeekedFunction() const
  */
 void RizinDatabase::fetchFunctionsAndGlobals(Config &rzconfig) const
 {
-	auto list = rz_analysis_get_fcns(_rzcore.analysis);
+	auto list = rz_analysis_function_list(_rzcore.analysis);
 	if (list != nullptr) {
 		FunctionContainer functions;
 		for (RzListIter *it = list->head; it; it = rz_list_iter_get_next(it)) {

